### PR TITLE
CON-1241: Updated Python API README to point to the PyTorch sample

### PR DESCRIPTION
**Pull request by shamsasari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,12 +217,12 @@ good compatibility with existing Python libraries.
...
 As work in progress, there are plently of issues and features missing, which we plan to address. Some of which are:
 
-* Only simulation mode works. Mock mode should also work, but there's no way to interact with the Jep interpreter 
-  without using reflection.
-* All the necessary tool, such as Python, pip and Gramine, must be installed locally
+* Mock mode support is limited. There's currently no way to inspect objects from the Python environment without 
+  using reflection.
+* All the necessary tools, such as Python, pip and Gramine, must be installed locally
 * Most likely the enclave will only work on the same machine that it was built on.
 * Only a single Python file is supported
```
Comment (anien) [inline README.md:224]: Two full stops missing at the end of the sentence. Would be nice to have them for consistency.

File: README.md [hunk 2]
```diff
@@ -217,12 +217,12 @@ good compatibility with existing Python libraries.
...
-
+* There's no yet API to send responses to other than the requester.
```
Comment (anien) [inline README.md:225]: _There's no API yet_ sounds better to me.

**Comment by shamsasari** [comment]
Thanks, PTAL.

**Review by anien — APPROVED** [review]
Nice, thanks.

**Pull request merged**